paddle/fluid/framework/selected_rows.h

```diff
@@ -133,9 +183,10 @@ class SelectedRows {
   // SelectedRows are simply concated when adding together. Until a
   // SelectedRows add a Tensor, will the duplicate rows be handled.
   Vector<int64_t> rows_;
-  std::unordered_map<int64_t, int64_t> id_to_index_;
+  std::unordered_map<int64_t, int64_t>
+      id_to_index_;  // should not be used when ids has duplicate member
```

Comment:
What is `ids` ???

Reply:
it means rows_ has duplicate member and ids indicate word id, and member means each id in ids